inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -48,14 +50,15 @@ In order to estimate stunting in the surveyed population, we looked at Height fo
 
 * Stunting: HAZ score <-2;
 * Moderate stunting: HAZ score >=-3 and <-2; Severe stunting: HAZ score <-3.
-* Exclusion of z-scores from Observed mean SMART flags included: WHZ -3 to 3; HAZ -3 to 3; WAZ -3 to 3.
+* Exclusion of z-scores from Observed mean SMART flags included: WHZ -3 to 3; HAZ -3 to 3; (Weight for age Z-scores) WAZ -3 to 3.
+
 
 # Results
 
 ```{r}
 # read in data
 # you can use rio to read in tabular data (Excel/csv) files
-survey_data <- rio::import("<your_data>")
+survey_data <- rio::import("<your_data>", which = 2)
 
```

Comment:
need to add a section on weighting and clustering with svy design. 
have emailed you an example of weighting shite - just in case 
see my comment on mortality [here ](https://github.com/R4EPI/sitrep/pull/84/files#r261505677)